web/impact/impact/v1/helpers/judging_round_helper.py

```diff
@@ -48,3 +49,8 @@ def full_name(self):
         # short_name is "2017-04 BOS CH IL Round 1".  See ticket
         # AC-5694 for proposed cleanup.
         return self.subject.short_name()
+
+    @property
+    def has_criteria(self):
+        criteria = JudgingRound.objects.get(pk=self.subject.id).criterion_set.all()
+        return any(criteria)
```

Comment:
better to use `JudgingRound.objects.get(pk=self.subject.id).criterion_set.count() > 0` rather than evaluating the queryset to see if anything is in it.